OracleAccessManagement/dockerfiles/12.2.1.4.0/README.md

```diff
@@ -0,0 +1,447 @@
+Building an Oracle Access Management Image using Dockerfile Samples
```

Comment:
You need to remove as many references to Docker as possible. These files create container images. Docker is optional.